tests/scripts/deploy_admission_controller.sh

```diff
@@ -3,24 +3,59 @@
 # Sets up the environment for the admission controller webhook in the active cluster.
 set -eo pipefail
 
+trap cleanup EXIT SIGINT ERR
+function cleanup() {
+  kubectl -n rook-ceph delete validatingwebhookconfigurations $WEBHOOK_CONFIG_NAME
```

Comment:
You might want to `|| true` to not circle back in an endless trap.

Reply:
added.